build-configuration/android-library.gradle

```diff
@@ -0,0 +1,84 @@
+/**
+ * Base build configuration for all android libraries (modules applying the android library plugin).
+ *
+ * Don't add any configuration here that is not common to *every* android library.
+ */
+apply plugin: "com.android.library"
+apply plugin: "kotlin-android"
+apply plugin: "org.jetbrains.dokka"
+
+apply from: rootProject.file('build-configuration/detekt.gradle')
+
+assemble.dependsOn('lint')
+check.dependsOn('checkstyle')
+
+configurations {
+    javadocDeps
+    ktlint
+}
+
+if (System.getenv("JITPACK")) {
+    group='com.github.stripe.stripe-android'
+}
+
+android {
```

Comment:
thanks for adding this! Is it possible for users to override `android {}` values?

Reply:
Yes! `stripe-core` now [does that](https://github.com/stripe/stripe-android/pull/4655/files#diff-4213b3c50027f9be3e6ec8b126f19697a5b23aef63d700c64bdfce826753e6f9R14-R22). Gradle's closures called several times are aggregated.